packages/garbo/src/outfit/barf.ts

```diff
@@ -160,5 +164,5 @@ export function barfOutfit(spec: OutfitSpec = {}, sim = false): Outfit {
     parka: "kachungasaur",
   });
 
-  return outfit;
+  return { outfit, extraValue };
```

Comment:
I don't love doing this for barf and nothing else, seems counterintuitive & inconsistent. But I get it

Might be worth exporting two functions, like the `freeFightFamiliarData` and `freeFightFamiliar` thing?